### PR TITLE
Added calsys scripts & laser testing scripts

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: python/lsst/ts/externalscripts/characterization/scripts.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import numpy as np
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/characterization/scripts.py]: Please, rename this file to reflect what is inside it. I suggest `laser_characterization.py`

File: python/lsst/ts/externalscripts/coordination/script.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
+__all__ = ["LaserCoordination"]
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/coordination/script.py]: Please, rename file to reflect what is inside. I suggest `laser_coordination.py`

File: scripts/auxtel/calsys_take_narrowband_data.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+from lsst.ts.externalscripts.auxtel import CalSysTakeNarrowbandData
+
+CalSysTakeNarrowbandData.main()
```
Comment (tribeiro) [inline scripts/auxtel/calsys_take_narrowband_data.py]: should probably add the standard 

```
if __name__ == `__main__`:
    CalSysTakeNarrowbandData.main()
```

File: scripts/auxtel/calsys_takedata.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+from lsst.ts.externalscripts.auxtel import CalSysTakeData
+
+CalSysTakeData.main()
```
Comment (tribeiro) [inline scripts/auxtel/calsys_takedata.py]: should probably add the standard

```
if __name__ == "__main__":
    CalSysTakeData.main()
```

File: scripts/main/laser_coordination.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+
+from lsst.ts.externalscripts.coordination import LaserCoordination
+LaserCoordination.main()
```
Comment (tribeiro) [inline scripts/main/laser_coordination.py]: should probably add the standard

```
if __name__ == "__main__":
    LaserCoordination.main()
```

File: python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+import SALPY_ATArchiver
+import csv
+import datetime
+import os
+import pathlib
+import requests
+
+__all__ = ["CalSysTakeNarrowbandData"]
+
+def is_sequence(value):
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py]: import from `python/lsst/ts/externalscripts/auxtel/calsys_takedata.py`

File: python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py [hunk 7]
```diff
@@ -0,0 +1,376 @@
...
+    """Return True if value is a sequence that is not a `str` or `bytes`.
+    """
+    if isinstance(value, str) or isinstance(value, bytes):
+        return False
+    return isinstance(value, collections.Sequence)
+
+
+def as_array(value, dtype, nelt):
```
Comment (tribeiro) [inline python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py]: import from python/lsst/ts/externalscripts/auxtel/calsys_takedata.py

File: python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py (hunk 6 above)
Comment (couger01) [reply python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py]: I'm not sure what you're trying to say here.

Comment (tribeiro) [reply python/lsst/ts/externalscripts/auxtel/calsys_take_narrowband_data.py]: The method `is_sequence` is also defined in `python/lsst/ts/externalscripts/auxtel/calsys_takedata.py`. That means you should be able to do 

`from lsst.ts.externalscripts.auxtel.calsys_takedata import is_sequence`

same for `as_array` later.

**Pull request merged**